modules/codemod/package.json

```diff
@@ -11,7 +11,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Workday/canvas-kit.git"
+    "url": "https://github.com/workday/canvas-kit.git",
```

Comment:
Why not https://github.com/workday/canvas-kit? https://github.com/workday/canvas-kit.git will redirect to https://github.com/workday/canvas-kit, so why not save the user a redirection?

Reply:
Going by npmjs.com docs and examples. I think this field can get picked up for other purposes (VCS tools in editors maybe) so they recommend the `.git` extension, but not the `git+` prefix. I honestly don't have a specific example of how this matters though.

Reply:
From *Additional References* on the PR description:
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository
https://docs.npmjs.com/cli/v8/commands/npm-repo

https://github.com/facebook/react/blob/main/packages/react/package.json#L32
https://github.com/chakra-ui/chakra-ui/blob/main/packages/react/package.json#L74

Reply:
I didn't see any documentation referencing the `.git` extension in the URL, but I do see their examples using it. I know redirects can delay the page from rendering on higher latency connections which is why I like to avoid them.

Reply:
I agree this is not properly explained. They just say:
> The URL should be a publicly available (perhaps read-only) url that can be handed directly to a VCS program without any modification. It should not be a url to an html project page that you put in your browser. It's for computers.

React uses .git, Chakra UI does not. I confirmed on npmjs.com that it will NOT include the .git extension for the repository URL. It also ignores the directory, and goes straight to the main repo page on GitHub. I had trouble confirming whether it leaves off `.git` when using `npm repo` but I did observe that it generates a URL when a directory field is specified, so I highly suspect it will also leave off .git. I just couldn't absolutely confirm it leaves off .git, but it will generate a link like: `https://github.com/chakra-ui/chakra-ui/tree/master/packages/breadcrumb` which redirects to `https://github.com/chakra-ui/chakra-ui/tree/main/packages/breadcrumb`. That leads me to suspect `.git is probably left off altogether when it generates this URL.

If we want a human-oriented URL then we can add a homepage URL to each package. I left off this improvement for now.

TL;DR: I observed no redirects akin to https://github.com/workday/canvas-kit.git to https://github.com/workday/canvas-kit in testing, and I would feel safer leaving `.git` there for possible VCS (version control systems) that might rely on it to work seamlessly. I don't feel too strongly though, so if you still insist on taking off `.git` I do not object and I'm happy to make that change.